projects/components/src/confirmation/confirmation-modal.component.ts

```diff
@@ -51,5 +56,6 @@ export interface ConfirmationModalData {
   cancelButtonLabel?: string;
   confirmButtonLabel?: string;
   confirmButtonRole?: ButtonRole;
-  descriptionText: string;
+  descriptionText?: string;
```

Comment:
See comment [here](https://github.com/hypertrace/hypertrace-ui/pull/424#discussion_r541949306) about api design, this has a similar issue. If I provide both text and content, it's unclear which takes precedence. Instead, we should take them both in as a union arg so only one or the other can be provided. To do that, we likely need to make a breaking change because `descriptionText` doesn't make sense for a template, but since we're not actually publishing this we can just update the consumers (otherwise, I'd make descriptionText optional and deprecated).